### PR TITLE
feat: Save Auth state

### PR DESCRIPTION
**Pull request by souyahia-monk**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -63,7 +64,7 @@ function App() {
         await Font.loadAsync(customFonts);
         // Artificially delay for two seconds to simulate a slow loading
         // experience. Please remove this if you copy and paste the code!
-        await new Promise((resolve) => { setTimeout(resolve, 2000); });
+        // await new Promise((resolve) => { setTimeout(resolve, 2000); });
```
Comment (natsukiai) [inline src/components/App.js:67]: Actually, I was liking the first loader saying "Launching the app" ^^ but as the comment says, we can disable it.

File: src/screens/Landing/SignOut/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  return (
+    <Menu
+      visible={visible}
+      onDismiss={closeMenu}
+      anchor={(
+        <List.Subheader style={styles.subheader}>
+          <IconButton icon="account-circle" onPress={openMenu} size={32} />
+          You are connectedsq
+        </List.Subheader>
```
Comment (natsukiai) [inline src/screens/Landing/SignOut/index.js:44]: Here's a typo ^^

File: src/screens/Landing/SignOut/index.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      )}
+      style={{ zIndex: 999 }}
```
Comment (natsukiai) [inline src/screens/Landing/SignOut/index.js:46]: Is this z-index mandatory ? I'm really not a fan of absolute z-index.

File: src/components/App.js (hunk 1 above)
Comment (souyahia-monk) [reply src/components/App.js:67]: Actually, I forgot to comment back this line. I just commented it for faster testing, should I add it back ?

Comment (natsukiai) [reply src/components/App.js:67]: Yes please. :-)

**Review by natsukiai — APPROVED** [review]

**Pull request merged**